openwrt/configs/common.config

```diff
@@ -71,7 +78,6 @@ CONFIG_PACKAGE_zabbix-agentd=y
 CONFIG_PACKAGE_zabbix-extra-mac80211=y
 CONFIG_PACKAGE_zabbix-extra-network=y
 CONFIG_PACKAGE_zabbix-extra-wifi=y
-CONFIG_PACKAGE_zabbix-sender=y
```

Comment:
I saw the note about removing zabbix server flag but this is zabbix sender. Intended?

Reply:
Ill double check, but IIRC this was trigger us to build all zabbix server deps

Reply:
That makes sense. My only concern is that @davidelang uses it. Might be nothing.
https://linux.die.net/man/1/zabbix_sender